### PR TITLE
feat(vps): add APAC plan codes for upgrade

### PR DESCRIPTION
**Pull request by akuracy**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-4961
| License          | BSD 3-Clause

## Description

Ensure APAC plan codes will be used during upgrade.
Needed before migrating all APAC models.

### CODE REVIEW

File: packages/manager/modules/vps/src/upgrade/vps-upgrade.controller.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ export default class VpsUpgradeCtrl {
       modelVersion,
       versionInfos.year < 2018 ? '2018v3' : '2018v4',
     );
-    const mappedType = get(OFFER_AGORA_MAPPING, modelType, modelType);
+    const mappedType = get(OFFER_AGORA_MAPPING, modelType, modelType == 'ssd' && (modelVersion == "2017v1" || modelVersion == "2018v2") ? 'ssd-discovery' : modelType);
```
Comment (antleblanc) [inline packages/manager/modules/vps/src/upgrade/vps-upgrade.controller.js:90]: Retrieving the type is getting harder.

Why not considering having a method in a service for this?

File: packages/manager/modules/vps/src/upgrade/vps-upgrade.controller.js [hunk 2]
```diff
@@ -258,7 +258,7 @@ export default class VpsUpgradeCtrl {
                 name: this.stateVps.model.offer,
                 details: VpsUpgradeCtrl.findMatchingUpgradeOffer(
                   this.stateVps.offerType,
-                  this.stateVps.model.name,
+                  this.a.model.name,
```
Comment (antleblanc) [inline packages/manager/modules/vps/src/upgrade/vps-upgrade.controller.js:261]: I suspect a typo. I think we can keep it as:

```suggestion
                  this.stateVps.model.name,
```

Comment (akuracy) [reply packages/manager/modules/vps/src/upgrade/vps-upgrade.controller.js:261]: Yes, thank you, fixing it

**Comment by antleblanc** [comment]
I've added the internal ticket reference into the Pull Request description table.

**Pull request merged**